packages/proxmark3/build.sh

```diff
@@ -5,6 +5,14 @@ TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
 TERMUX_PKG_VERSION="1:4.13441"
 TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
 TERMUX_PKG_SHA256=49c4f1854b364aa7ea7083581351f867128e71ea783d0ecd71fc41bcf7f63584
+TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
+TERMUX_PKG_BUILD_IN_SRC="true"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
```

Comment:
why is this repeated again in lines 16 17 and 18??

Reply:
Because apparently I can't copy and paste properly